mlir/test/Dialect/MIOpen/lowering_top_level.mlir

```diff
@@ -67,10 +67,15 @@ func @miopen_conv2d_bwd_data(%filter : memref<1x128x8x3x3xf32>, %input : memref<
 // CHECK-LABEL: func {{@miopen_conv2d_bwd_data.*%arg0.*%arg1.*%arg2}}
 // CHECK-NOT:   miopen.conv2d_bwd_data
 // CHECK-NEXT:  miopen.transform(%arg0)
+// CHECK-NEXT:  miopen.transform
```

Comment:
please amend these newly added `miopen.transform` to check the operations added

Reply:
@ltqin please help improve checks here, use regex to check the attributes applied to `miopen.transform`.

Reply:
@whchung done, please check that this is what you want